docs/include/filter.asciidoc

```diff
@@ -0,0 +1,171 @@
+[id="plugins-{type}s-common-options"]
+==== Common Options
+
+The following configuration options are supported by all filter plugins:
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Setting |Input type|Required
+| <<plugins-{type}s-{plugin}-add_field>> |<<hash,hash>>|No
+| <<plugins-{type}s-{plugin}-add_tag>> |<<array,array>>|No
+| <<plugins-{type}s-{plugin}-enable_metric>> |<<boolean,boolean>>|No
```

Comment:
@jordansissel but we don't include a base plugin doc? The filter/codec/input/output asciidoc content is a merge of what was originaly from the **plugin.rb base class** (where `enable_metric` is coming from) and the **plugin type base class**.